models/issue_label.go

```diff
@@ -98,7 +98,11 @@ func (label *Label) ForegroundColor() template.CSS {
 
 // NewLabels creates new label(s) for a repository.
 func NewLabels(labels ...*Label) error {
-	_, err := x.Insert(labels)
```

Comment:
`x.Insert(slice)` is one INSERT command, `x.Insert(slice...)` will generate N INSERT command. A transaction for a better performance.